### PR TITLE
added add_two method

### PR DESCRIPTION
**Pull request by aposheker**
Added add_two method. Input has to be a number.

### CODE REVIEW

File: goat/placeholder.py [hunk 1]
```diff
@@ -20,3 +20,20 @@ def multiply_by_2(x: int) -> int:
     if type(x) == int:
         return x * 2
     raise TypeError("The type of the input wasn't int.")
+    
+def add_two(x):
```
Comment (SeyhmusGuler) [inline goat/placeholder.py]: I think you should add typehints to the function.

**Comment by SeyhmusGuler** [comment]
Could you please add a description to the PR?

Also the function you added doesn't have any tests; could you add tests for it?

**Comment by SeyhmusGuler** [comment]
looks good.

**Review by SeyhmusGuler — APPROVED** [review]

File: goat/placeholder.py (hunk 1 above)
Comment (aposheker) [reply goat/placeholder.py]: Typehint is added.

**Pull request merged**